### DISCUSSION TITLE
Update version with the same config

### DISCUSSION THREAD

**Discussion by wangcongyi** [swc-project/swc#4390] [Q&A]
I used @swc/core v1.2.107 to replace the babel-loader several months ago.  It's worked.
but today i update @swc/core v1.2.170, Something wrong

![image](https://user-images.githubusercontent.com/13843979/164424362-f0e4139c-eb98-40fe-acb2-928457456958.png)

the `.swcrc`  file is the same
```js

[
  {
    "test": ".tsx?$",
    "jsc": {
      "parser": {
        "syntax": "typescript",
        "tsx": true,
        "decorators": true,
        "dynamicImport": true
      },
      "transform": {
        "react": {
          "refresh": true
        }
      }
    }
  },
  {
    "test": ".jsx?$",
    "jsc": {
      "parser": {
        "syntax": "ecmascript",
        "jsx": true,
        "decorators": true,
        "dynamicImport": true
      },
      "transform": {
        "react": {
          "refresh": true
        }
      }
    }
  }
]

```

Did I miss something config? I can't find the doc.

**Comment by kdy1** [comment]
`.jsx?$` and `.tsx?$` seems wrong to me.
It's regex, so it should be `\\.jsx?$` I think?
I'm not sure why it worked previously. Maybe quirks of old version of regex crates

**Reply by wangcongyi** [reply]
According to the `swc-loader` [doc.](https://swc.rs/docs/configuration/compilation#test)
Is the doc. not updated yet?

**Reply by wangcongyi** [reply]
and i use `\\.jsx?$`, it's still wrong.

@swc/core": "^1.2.171"

**Reply by kdy1** [reply]
Can you provide some repro case?

**Reply by kdy1** [reply]
I added the test for it and it worked. See https://github.com/swc-project/swc/pull/4398

**Comment by kdy1** [comment]
Ah, you may need to remove `dynamicImport`

**Comment by kdy1** [comment]
Also, https://github.com/swc-project/swc/pull/4236 can be the cause

**Comment by wangcongyi** [comment]
After many attempts, I didn't remove `dynamicImport`. and it's worked.
so I think `@swc/core` still has bugs.
By the way, the swc-loader is so faster than babel-loader. 
I love it, tks for your work
